src/bodies/Bodies.jl

```diff
@@ -0,0 +1,29 @@
+module Bodies
+
+import Base: parent
+
+export CelestialBody, SolarSystemBarycenter, ssb, Sun, sun, parent, naifid
+
+abstract type CelestialBody end
+
+abstract type Barycenter <: CelestialBody end
+
+struct ParentOfSSB <: Barycenter end
+
+struct SolarSystemBarycenter <: Barycenter end
+const ssb = SolarSystemBarycenter()
+Base.show(io::IO, ::SolarSystemBarycenter) = print(io, "Solar System Barycenter")
+parent(::SolarSystemBarycenter) = ParentOfSSB()
+naifid(::SolarSystemBarycenter) = 0
+
+struct Sun <: CelestialBody end
+const sun = Sun()
+Base.show(io::IO, ::Sun) = print(io, "Sun")
```

Comment:
Wouldn't something like
```julia
Base.show(io::IO, x::CelestialBody) = print(io, string(typeof(x)))
```
for most cases?

Reply:
Good idea 👍 That works except for the barycenters which I want to print as „Earth Barycenter“ etc.

Reply:
Yes, I've seen a case where this wouldn't work, but you can overload it :slightly_smiling_face: